config/boards/armsom-w3.wip

```diff
@@ -0,0 +1,29 @@
+# Rockchip RK3588 SoC octa core 8-32GB SoC 2.5GBe PoE eMMC USB3 NvME
+BOARD_NAME="ArmSoM W3"
+BOARDFAMILY="rockchip-rk3588"
+BOARD_MAINTAINER="armsom-team"
+BOOTCONFIG="rock-5b-rk3588_defconfig"
```

Comment:
It's using rock5b's uboot, does it support features like spi nvme boot?

Reply:
We have checked the uboot code of radax, which also supports nvme boot. There is no spi boot but it does not affect